### PR TITLE
COM-533 Update Report Logic when checking for Viral Load Exams

### PR DESCRIPTION
**Pull request by prashantsable1991**
Updated Report Logic when checking for new viral load Exams.

### CODE REVIEW

File: sql/report_util_functions.sql [hunk 1]
```diff
@@ -592,8 +592,12 @@ CREATE FUNCTION retrieveViralLoadTestDate(
     p_patientId INT(11)) RETURNS DATE
     DETERMINISTIC
 BEGIN
-    DECLARE viralLoadTestDateUuid VARCHAR(38) DEFAULT 'cac6bf44-f671-4f85-ab76-71e7f099d3cb';
-    DECLARE viralLoadTestUuid VARCHAR(38) DEFAULT '4d80e0ce-5465-4041-9d1e-d281d25a9b50';
+    DECLARE routineViralLoadTestDateUuid VARCHAR(38) DEFAULT '9ee13a14-c7ce-11e9-a32f-2a2ae2dbcce4';
+    DECLARE routineViralLoadTestUuid VARCHAR(38) DEFAULT 'e91915f5-bfda-42ca-bd03-25d2810ee82e';
+    DECLARE targetedViralLoadTestDateUuid VARCHAR(38) DEFAULT '9ee13e38-c7ce-11e9-a32f-2a2ae2dbcce4';
+    DECLARE targetedViralLoadTestUuid VARCHAR(38) DEFAULT 'ac479522-c891-11e9-a32f-2a2ae2dbcce4';
+    DECLARE notDocumentedviralLoadTestDateUuid VARCHAR(38) DEFAULT '9ee140e0-c7ce-11e9-a32f-2a2ae2dbcce4';
```
Comment (tmvumbi2) [inline sql/report_util_functions.sql]: @prashantsable1991 please use a capital 'V' for viral here: i.e. `notDocumentedViralLoadTestDateUuid`.
Same feedback for the line below (i.e. `notDocumentedViralLoadTestUuid`)

File: sql/report_util_functions.sql [hunk 2]
```diff
@@ -1135,8 +1139,12 @@ CREATE PROCEDURE retrieveViralLoadTestDateAndResult(
     )
     DETERMINISTIC
 proc_vital_load:BEGIN
-    DECLARE viralLoadTestDateUuid VARCHAR(38) DEFAULT 'cac6bf44-f671-4f85-ab76-71e7f099d3cb';
-    DECLARE viralLoadTestUuid VARCHAR(38) DEFAULT '4d80e0ce-5465-4041-9d1e-d281d25a9b50';
+    DECLARE routineViralLoadTestDateUuid VARCHAR(38) DEFAULT '9ee13a14-c7ce-11e9-a32f-2a2ae2dbcce4';
+    DECLARE routineViralLoadTestUuid VARCHAR(38) DEFAULT 'e91915f5-bfda-42ca-bd03-25d2810ee82e';
+    DECLARE targetedViralLoadTestDateUuid VARCHAR(38) DEFAULT '9ee13e38-c7ce-11e9-a32f-2a2ae2dbcce4';
+    DECLARE targetedViralLoadTestUuid VARCHAR(38) DEFAULT 'ac479522-c891-11e9-a32f-2a2ae2dbcce4';
+    DECLARE notDocumentedviralLoadTestDateUuid VARCHAR(38) DEFAULT '9ee140e0-c7ce-11e9-a32f-2a2ae2dbcce4';
```
Comment (tmvumbi2) [inline sql/report_util_functions.sql]: @prashantsable1991 please use a capital 'V' for viral here: i.e. `notDocumentedViralLoadTestDateUuid`.
Same feedback for the line below (i.e. `notDocumentedViralLoadTestUuid`)

**Comment by tmvumbi2** [comment]
@prashantsable1991 Please retire the old concepts `Viral Load` (uuid: `4d80e0ce-5465-4041-9d1e-d281d25a9b50`) and `Viral Load Test Date` (uuid: `cac6bf44-f671-4f85-ab76-71e7f099d3cb`)

**Comment by tmvumbi2** [comment]
@prashantsable1991 please delete the skip logic for `Viral Load` (in cameroon-bahmni-config)

**Comment by tmvumbi2** [comment]
@prashantsable1991 We need to update the logic below `-- select the test date to use`: if both openelis and form are present, use openelis

**Comment by prashantsable1991** [comment]
@tmvumbi2 Thanks for the review.

**Pull request merged**